File: TD/TD3/Code/2.cpp
/* Retourner la moyenne de deux réels A et B donnée en paramètres */
#include <iostream>

float retour(float a, float b)
{
  float c;
  
  c = (a + b) / 2;
  
  return c;
}

int main()
{
  float a;
  float b;
  
  std::cout << "Donne moi premier réel : " << std::flush;
  std::cin >> a;
  std::cout << "Donne moi deuxième réel : " << std::flush;
  std::cin >> b;
  
  std::cout << "La moyenne est " << retour(a, b) << std::endl;
  
  return 0;
}

File: TD/TD3/Code/Python/2.py
#!/usr/bin/python
# Retourner la moyenne de deux réels A et B donnée en paramètres


def retour(a, b):
    return (a + b) / 2


print('Donne moi la première valeur : ', end='')
a = float(input())
print('Donne moi la deuxième valeur : ', end='')
b = float(input())

print('La moyenne de ces deux valeurs est', retour(a, b))
